contracts/chain-adapters/Arbitrum_Adapter.sol

```diff
@@ -154,18 +154,27 @@ contract Arbitrum_Adapter is AdapterInterface, CircleCCTPAdapter {
     // L2 Gas price bid for immediate L2 execution attempt (queryable via standard eth*gasPrice RPC)
     uint256 public constant L2_GAS_PRICE = 5e9; // 5 gWei
 
+    // Native token expected to  be sent in L2 message. Should be 0 for only use case of this constant, which
+    // includes is sending messages from L1 to L2.
```

Comment:
I don't really understand this comment. I think it refers to the amount of the native token to be sent, but I'm not sure.

```suggestion
    // Native token expected to be sent in L2 message. Should be 0 for only use case of this constant, which
    // includes is sending messages from L1 to L2.
```